.env.example

```diff
@@ -170,9 +170,9 @@ export BOOKWAREHOUSE_NAMESPACE=bookwarehouse
 # Default: 90s
 # export TIMEOUT=90s
 
-### optional: Whether to publish OSM docker images to container registry
```

Comment:
You should still keep the PUBLISH_IMAGES flag. It's separate from whether or not a private registry is being used

Reply:
I was considering that but we don't use it anywhere in the demo - should I still keep it?